cpp/src/aztec/honk/proof_system/prover.cpp

```diff
@@ -332,27 +333,37 @@ template <typename settings> void Prover<settings>::execute_univariatization_rou
     // are not needed in the proof itself.
 
     // Construct MLE opening point
+    // Note: for consistency the evaluation point must be constructed as u = (u_d,...,u_1)
     for (size_t round_idx = 0; round_idx < key->log_n; round_idx++) {
-        std::string label = "u_" + std::to_string(round_idx + 1);
+        std::string label = "u_" + std::to_string(key->log_n - round_idx);
```

Comment:
This meant that the evaluations in Gemini were being done at $(u_d, ..., u_1)$ rather than $(u_1, ..., u_d)$.